implementation/contracts/system/TBTCToken.sol

```diff
@@ -53,12 +53,13 @@ contract TBTCToken is ERC20Detailed, ERC20, VendingMachineAuthority {
     /// @param _spender   Address of contract authorized to spend.
     /// @param _value     The max amount they can spend.
     /// @param _extraData Extra information to send to the approved contract.
-    function approveAndCall(address _spender, uint256 _value, bytes memory _extraData) public returns (bool success) {
+    function approveAndCall(address _spender, uint256 _value, bytes memory _extraData) public returns (bool) {
         tokenRecipient spender = tokenRecipient(_spender);
         if (approve(_spender, _value)) {
             spender.receiveApproval(msg.sender, _value, address(this), _extraData);
             return true;
         }
+        return false;
```

Comment:
This structure seems different---do we need to introduce the same `if (approve...)` guard to the TDT?

Reply:
`approve()` does not `return bool` for ERC721

Reply:
👍